controlplane/kubeadm/controllers/controller.go

```diff
@@ -410,8 +410,8 @@ func (r *KubeadmControlPlaneReconciler) reconcileDelete(ctx context.Context, clu
 
 	// Ignore the health check results here as well as the errors, they are used to set health related conditions on Machines.
 	// Errors may be dues not being able to get workload cluster nodes.
-	r.managementCluster.TargetClusterControlPlaneHealthCheck(ctx, controlPlane, util.ObjectKey(cluster))
-	r.managementCluster.TargetClusterEtcdHealthCheck(ctx, controlPlane, util.ObjectKey(cluster))
+	r.managementCluster.TargetClusterControlPlaneHealthCheck(ctx, controlPlane, util.ObjectKey(cluster)) //nolint
+	r.managementCluster.TargetClusterEtcdHealthCheck(ctx, controlPlane, util.ObjectKey(cluster)) //nolint
```

Comment:
Can we ignore the error explicitly, or just log them?

Reply:
Logged.